geometries/CMS_Phase2/Pixel/Pixel_V4/Pixel_V4_1_2_1.cfg

```diff
@@ -0,0 +1,26 @@
+
+Tracker Pixels {
+
+  phiSegments 4
+  etaCut 10
+  zError 70
+
+  smallDelta 0
+  bigDelta 3
+
+  servicesForcedUp false
+
+  barrelRotation 1.57079632679
+  
+  @include-std CMS_Phase2/Pixel/moduleOperatingParms
+ 
+  trackingTags pixel,tracker
+  
+  barrelDetIdScheme Phase2Subdetector5
+  endcapDetIdScheme Phase2Subdetector4
+
+  @include BPIX_4_0_0.cfg
+  @include FPIX1_4_0_2_1.cfg
```

Comment:
I think you wanted to include the actual 4.1.2.1 config file here instead of 4.0.2.1 and 4.0.0.1, right?